perfkitbenchmarker/beam_benchmark_helper.py

```diff
@@ -18,40 +18,53 @@
 executions.
 """
 
+import fnmatch
 import os
+import logging
 
 from perfkitbenchmarker import dpb_service
 from perfkitbenchmarker import errors
 from perfkitbenchmarker import flags
 from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_benchmarks import beam_integration_benchmark
```

Comment:
This shouldn't be importing beam_integration_benchmark. b_i_b should be handling the details of which class to run; this shouldn't know anything about it.